src/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  font-weight: 300px;
+}
+
```

Comment:
you haven't added font-faces, [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 7th item